consumer/consumererror/partial.go

```diff
@@ -0,0 +1,28 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package consumererror // import "go.opentelemetry.io/collector/consumer/consumererror"
+
+type Partial struct {
```

Comment:
Can you add godoc comments for this error?

Reply:
Good catch, added.